### ISSUE TITLE
[AXE-CORE]: Outreach & Events - Links must have discernible text

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#23206]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix issue.
1. At message the accessibility specialist when issue is resolved.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement
Accessibility specialists discovered Nightwatch axe checks failing silently during a recent staging review. Multiple axe violations were discovered where empty link tags were on the pages. These are problematic because screen readers will announce "Link" but not what the link is. 

## Tasks
- [ ] If there is no value or HREF, the link `<a></a>` should be removed. 

## Environments
- https://www.va.gov/outreach-and-events/events/accessing-covid-19-vaccinations-at-the-va-what-you-need-to-know/

## Acceptance Criteria
- [ ] Zero axe-core violations are reported for links with missing text

## Screenshots
![Screen Shot 2021-04-14 at 10 41 51 AM](https://user-images.githubusercontent.com/934879/114739198-4bb22080-9d0e-11eb-8e9d-2cc8ba54a9a0.png)

**Comment by brianalloyd** [comment]
@RLHecht we reviewed this in grooming and believe this is something the `Sitewide Content` team would handle, can you review and let me know your thoughts.

**Comment by RLHecht** [comment]
Thanks @brianalloyd This is another issue in Outreach and Events, which Sitewide Content doesn't work on. If it's content and not FE, I'm not sure who the appropriate person would be. It looks like it's just deleting the extra link before the word.

**Comment by timwright12** [comment]
Looks like this was fixed in a recent content deployment: https://www.va.gov/outreach-and-events/events/accessing-covid-19-vaccinations-at-the-va-what-you-need-to-know/

**Comment by 1Copenut** [comment]
Two small `<a>` tags are still showing without text. When those two are removed, we'll be good to go.

**Comment by timwright12** [comment]
@1Copenut ok, I think we're good now

**Comment by 1Copenut** [comment]
@timwright12 Verified in prod, zero issues showing in axe browser plugin. Moving to close.
